frontend/packages/ceph-storage-plugin/src/utils/create-storage-system.ts

```diff
@@ -26,8 +26,8 @@ export const getSupportedVendors = (csv: ClusterServiceVersionKind): string[] =>
 export const getStorageSystemKind = ({ kind, apiVersion, apiGroup }) =>
   `${kind.toLowerCase()}.${apiGroup}/${apiVersion}`;
 
-export const getExternalSubSystemName = (name: string = '') =>
-  name.toLowerCase().replace(/\s/g, '-');
+export const getExternalSubSystemName = (name: string = '', storageClassName: string = '') =>
```

Comment:
```suggestion
export const getExternalSubSystemName = (name: string = '', storageClassName: string) =>
```
In case storage class names are always present, please make the function signature not default to anything.

Reply:
ack